### DISCUSSION TITLE
Columns Re Ordering without having to manually select each one

### DISCUSSION THREAD

**Discussion by lcslouis** [Budibase/budibase#3674] [Ideas]
Hello on List pages Can a better way of reordering the columns be implemented? 

Right now process is go to columns then select each column in the order in which i want to be displayed. It can be a time involved process if you have a lot of columns so being able to either drag and drop the column layout or apply an order priority number would be a big improvment. 

Also feedback: for restricting columns based on access right now I handle it by way of creating a screen based on the role assigned basic sees 1 screen and admins see another yes it is time consuming to do it that way but it's the only way right now.

**Comment by mjashanks** [comment]
Hi @lcslouis, we have a lot of much needed improvements to the table component: https://github.com/Budibase/budibase/discussions/1913

I have added your feedback and a link to this discussion into the link above. Thank you!
